src/test/python/testasfunction.py

```diff
@@ -0,0 +1,17 @@
+import find_projection
+import _pyrt
+import unittest
+
+class TestQueryInterface(unittest.TestCase):
+    def test_as_function(self):
+        propset = _pyrt.PropertySet()
+        propset.Insert("strmap", _pyrt.StringMap())
+        self.assertTrue(propset.HasKey("strmap"))
+        o = propset.Lookup("strmap")
+        strmap = _pyrt.StringMap._as(o)

```

Comment:
Semantics from _as seem backwards. As on comptr, etc. convert to a type specified in a template parameter. I would think...

`strmap = _pyrt.StringMap._from(o);`

… reads better. Thoughts?

Reply:
Yup, makes sense. Will change.